include/deal.II/lac/vector_memory.h

```diff
@@ -200,13 +200,13 @@ class VectorMemory : public Subscriptor
      * Move constructor: this creates a new Pointer by stealing the internal
      * data owned by @p p.
      */
-    Pointer(Pointer &&p) = default;
+    Pointer(Pointer &&) = default;
```

Comment:
I've long been bothered by this warning, but the problem is that if you remove the name, you can't refer to it in the documentation above any more :-(